programs/issue-3-speed-converter/run.py

```diff
@@ -0,0 +1,76 @@
+convert_dictionary = {
+ 1: {
+     "input": "m/s",
+     "output": "km/h",
+     "equation": "value * 3.6"},
+ 2: {
+     "input": "km/h",
+     "output": "m/s",
+     "equation": "value / 3.6"},
+ 3: {
+     "input": "mile/h",
+     "output": "km/h",
+     "equation": "value * 1.609"},
+ 4:  {
+     "input": "km/h",
+     "output": "mile/h",
+     "equation": "value / 1.609"},
+ 5: {
+     "input": "km/h",
+     "output": "% prędkości światła",
+     "equation": "value / 1.079e+9"}
+}
```

Comment:
dict formated in this way:

```
    convert_dictionary = {
        1: {
            "input": "m/s",
            "output": "km/h",
            "equation": "value * 3.6", # coma in last item
        },
        2: {
            "input": "km/h",
            "output": "m/s",
            "equation": "value / 3.6", # coma in last item
        }, # close bracket in new line
    }
```

produce much more dev-friendly diff:

```
        1: {
            "input": "m/s",
            "output": "km/h",
            "equation": "value * 3.6",
+           "new_option": "something", 
        },
        2: {
            "input": "km/h",
            "output": "m/s",
            "equation": "value / 3.6",
+           "new_option": "something",
        },
```

your old code would produce:
```
 5: {
     "input": "km/h",
     "output": "% prędkości światła",
-     "equation": "value / 1.079e+9"}
+     "equation": "value / 1.079e+9",
+     "new_option"}
}
```